app/views/dashboard_v2/_blog.html.erb

```diff
@@ -7,17 +7,17 @@
         <div aria-hidden = 'true' class="img" style="background-image: url('<%= @blog.scraped_image %>')">
       <% end %>
       <%# subscribe to blog updates possibly via email %>
-      <a class="ellipsis pull-right pl-2" data-toggle="dropdown" rel="tooltip" title="<%= translation('tag.show.blog_updates') %>" style="cursor:pointer; background-color: #444" aria-expanded="true"><i class="fa fa-ellipsis-h" style="color:#ccc;font-size:18px;margin-right:10px;"></i></a>
+      <a class="ellipsis pull-right pl-2" data-toggle="dropdown" rel="tooltip" title="<%= translation('tag.show.blog_updates',{},false) %>" style="cursor:pointer; background-color: #444" aria-expanded="true"><i class="fa fa-ellipsis-h" style="color:#ccc;font-size:18px;margin-right:10px;"></i></a>
       <div class="dropdown-menu">
         <div class="dropdown-item">
-          <a style="margin-top:-8px;" rel="tooltip" title="<%= translation('tag.show.blog_updates') %>" href="/subscribe/tag/blog">
+          <a style="margin-top:-8px;" rel="tooltip" title="<%= translation('tag.show.blog_updates',{},false) %>" href="/subscribe/tag/blog">
```

Comment:
Hi @imajit this is looking really nice! I wonder if it's possible to simplify the parameters and omit the `{}` by checking if `options == true` OR by using `translation(key, {html: true})` for some slightly shorter cleaner syntax?  This could definitely be done in follow-up, as this PR looks good. Are there any other places where this is used and where it would need to be updated?

Reply:
Yeah, I'll change the syntax to make it more clear. 
I think I have changed all the occurrences. If I find any new ones I'll make ftos to change them.